src/env.ts

```diff
@@ -1,3 +1,4 @@
+import 'dotenv/config'
```

Comment:
Not sure why this import is needed

Reply:
it's a really annoying mutation side-affect. Basically it populate `process.env` when you call this import